tests/explorer.tests/ComponentTests.cs

```diff
@@ -26,6 +27,7 @@ public async Task TestMinMaxRefinerComponent()
                 "gda_banking",
                 "loans",
                 "amount",
+                new ColumnInfo(DValueType.Integer, ColumnInfo.ColumnType.Regular),
```

Comment:
We should probably query for this (maybe in the ExplorerTestFixture, that we we can cache the data for all tests). Hard-coding the values is a bit error-prone. 
This is not a big deal though, we can log an issue and do it at a later date.

Reply:
I created the new issue #214 for this.